unittests/test-rootblock.cpp

```diff
@@ -54,15 +60,63 @@ TEST_CASE( "update inode + read/write to block device", "[Rootblock]" ) {
         REQUIRE(strcmp(firstInode->fileName,fileName) == 0);
         free(firstInode);
 
+        // ─────────────────────────────────────────────────────────────────
+        // Test last Inode
+
         //get last inode
         InodeStruct* lastInode = (InodeStruct *)malloc(BLOCK_SIZE);
-        //needs to be changed to LAST_INODE_BLOCK
-        lastInode = rootb->getInode(bd,FIRST_INODE_BLOCK+NUM_DIR_ENTRIES-1);
-        //test last inode in rootblock
+        /**
+         * getInode takes relative Values so NUMBER_OF_INODE_BLOCKS-1
+         * because Inodes start at 0
+         */
+        lastInode = rootb->getInode(bd,NUMBER_OF_INODE_BLOCKS-1);
+        // test last inode in rootblock
         REQUIRE(EMPTY_INODE_VALUE ==lastInode->firstDataBlock);
         REQUIRE(strcmp(lastInode->fileName,fileName) == 0);
         free(lastInode);
 
+        // ─────────────────────────────────────────────────────────────────
+        // Update Inode 33
```

Comment:
*Update Inode 32 oder Update the 33rd Inode

Reply:
dein ernst?

Reply:
Ja, so wie es aktuell da steht verwirrt es